packages/gasket-helper-intl/package.json

```diff
@@ -12,10 +12,9 @@
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "test": "npm run test:runner",
-    "test:runner": "mocha --require test/setup.js --require setup-env \"test/*.test.js\"",
-    "test:watch": "npm run test:runner -- --watch",
-    "test:coverage": "nyc --reporter=text --reporter=json-summary npm run test:runner",
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest",
```

Comment:
I see `NODE_OPTIONS='--unhandled-rejections=strict'` on several of these, and `--detectOpenHandles` on another. Is it possible to align on the jest flag? Or is that just a different behavior?

Reply:
`--detectOpenHandles` was used for future tests written. Given the nature of the test I felt it was appropriate. Without this flag you just timeout.